src/test/UniswapV3SplitRouteCallee.t.sol

```diff
@@ -0,0 +1,375 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2023 Maker Ecosystem Growth Holdings, INC.
+// Copyright (C) 2023 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
```

Comment:
I think this file has gone some auto formatting vs the 1inch test (e.g changing " -> ', spaces alignments etc).
Can we maybe revert that and if we think formatting fixes are needed it should be a separate PR that covers more than this file?

Reply:
I've tried to pinpoint the changes with the eyes, only noticed the quote differences.

Do you have any formatting process that you perform before PRs usually? e.g. some plugin in the VSCode that i could also run?

Or if you don't i would appreciate if you could enumerate some other issues you see with the formatting.

Reply:
Mmm we don't have an automatic formating process (but it looks like maybe you do).
I guess some of it is not only auto-formatting, but here is some stuff that is different than the 1Inch test (and makes the review process harder):

* import from `src` and not `..`
* having named vars in `function store`
* changing to `vat_` and not `vat` in VaultHolder constructor
* 1E18 => 1e18
* spacing in definition of GemAbstract, VatAbstract etc..
* spacing in definitions of VaultHolder and addresses below it
* spaces in `getLinkPriceRay`
* spaces in `frobMax`
* spaces in `testDrip`
* spaces in `testBarkLink`
* order of var definitions in `testTakeLinkUniswapSplitProfit`, `testTakeLinkUniswapSplitProfit`..

I guess some of these changes are good, but best to do them separately to ease the review.

Reply:
mm I don't think this has been handled.
Just adding a diff of the top of this file and the 1inch equivalent one to show what I mean:

![image](https://user-images.githubusercontent.com/115617270/226556921-259459fc-4a94-43be-8916-8588f146aaf1.png)

Reply:
oh, really sorry about this one 😓 , i probably pressed "Save" instead of "Save without formatting" at some point.

Should be either good or better now :)